arm/Microsoft.DataProtection/backupVaults/.parameters/parameters.json

```
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "adp-az-bv-min-001"
```

`adp` in CARML is only used as a prefix for 'dependency resources'. Further, the place where the `min` is used, we usually place the file's prefix. In case of a `min.parameter.json` we'd use `min`. In case of no prefix (i.e. only a `parameter.json`) we'd use `x` 

Here, we should use the following:

```suggestion
            "value": "<<namePrefix>>-az-bv-x-001"
```

---

Once changed, please refresh the readme using the `Set-ModuleReadMe` script

---

Yes. Missed that one.